classes/Validate.php

```diff
@@ -761,7 +761,7 @@ public static function isZipCodeFormat($zip_code)
      */
     public static function isOrderWay($way)
     {
-        return $way === 'ASC' | $way === 'DESC' | $way === 'asc' | $way === 'desc';
+        return $way === 'ASC' | $way === 'DESC' | $way === 'asc' | $way === 'desc' | $way === 'random';
```

Comment:
I don't understand why they use bitwise :facepalm: 
You should also consider `RANDOM` or test with strtolower like that
```suggestion
        $lowerWay = strtolower($way);
        return $lowerWay === 'asc' | $lowerWay === 'desc' | $lowerWay === 'random';
```